settings.yml

```diff
@@ -16,6 +16,11 @@ show_below_site_header:
   type: bool
   description: "Display the banner in the below site header connector."
 
+show_above_main_container:
+  default: false
+  type: bool
+  description: "Display the banner in the above main container connector, for sites with the sidebar enabled."
+
```

Comment:
I wanted to update to a select dropdown, but this would reset existing site's current settings.